website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -7,6 +7,13 @@ page_id: api-changelog
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-03-08
+
+* Updated [Workspaces API](/terraform/cloud-docs/api-docs/workspaces) to accept `vcs-repo.github-app-installation-id` to connect a workspace to a GitHub App Installation.
+* Updated [Registry Module API](/terraform/cloud-docs/api-docs/private-registry/modules) to accept `vcs-repo.github-app-installation-id` to connect to a GitHub App Installation.
+* Updated [Policy Sets API](/terraform/cloud-docs/api-docs/policy-sets) to accept `vcs-repo.github-app-installation-id` to connect to a GitHub App Installation.
+* Introduced the [GitHub App Installation APIs](/terraform/cloud-docs/api-docs/github-app-installations).
```

Comment:
Is there a formal order these are in already? If not it might be a good idea to put this bullet at the top so that readers will have context for the others.